DemoApp/Sources/Views/CallView/DemoCallContainerView.swift

```diff
@@ -33,6 +33,11 @@ internal struct DemoCallContainerView: View {
                     chatViewModel: chatViewModel
                 )
             )
+            .onCallEnded { call in
```

Comment:
we will eventually do a cookbook for this, right?

Reply:
Just added it